sandy/decay.py

```diff
@@ -233,12 +316,90 @@ def get_branching_ratio(self, with_uncertainty=True):
             else:
                 logging.warn(f"no branching ratio is found for {z}")
```

Comment:
I am not sure about keeping this warning. I am afraid it will pop up too many times.
It is not really useful, as you'll say what nuclide has branching ratios in the output dataframe